tools/bazel.rc

```diff
@@ -16,6 +16,10 @@ common:msvc --linkopt=/DEBUG:FULL
 
 test --test_output=errors
 
+# This isn't ideal, but with our multi-platform switching this is the only way
+# to make a bazel test ... useful.
```

Comment:
Can you put quotes around ``bazel test ...`` to differentiate the command from the rest of the line? Do tests invoked by ``bazel test foo/bar:baz`` still run, or only build with this? Would ``bazel test foo/bar:baz_linux`` work?

Reply:
Only :baz_linux :( still need to find a good way to do the aliasing. I may be able to have a custom repository rule generate a bzl that then is loaded with the rule to use, but haven't tried it yet.